.gitignore

```diff
@@ -1,6 +1,7 @@
 # .gitignore
 # don't push these files to the repository
 
+*.DS_Store
```

Comment:
What's that?

Reply:
Gets rid of a file automatically generated on the MacOS finder. Adding it in shouldn't effect anything, but I can take it out if you'd like me to.